packages/@react-spectrum/s2/src/TableView.tsx

```diff
@@ -779,20 +786,29 @@ function ResizableColumnContents(props: ResizableColumnContentProps) {
           </div>
           <Chevron size="M" className={chevronIcon} />
         </Button>
-        <Menu onAction={onMenuSelect} items={items} styles={style({minWidth: 128})}>
-          {(item) => <MenuItem>{item?.label}</MenuItem>}
+        <Menu onAction={onMenuSelect} styles={style({minWidth: 128})}>
+          {items.length > 0 && (
+            <MenuSection aria-label={stringFormatter.format('table.standardColumnMenu')}>
+              <Collection items={items}>
+                {(item) => <MenuItem>{item?.label}</MenuItem>}
+              </Collection>
```

Comment:
Looks like the sorting options aren't being included even with `isSortable` being true in this story: https://reactspectrum.blob.core.windows.net/reactspectrum/1b8dfffb43dab04ee2e289602d94ca176927bf5a/storybook-s2/index.html?path=/story/tableview--resizing-sortable-table&args=density:regular